environment.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+module Environment
+  class Depot
+    attr_accessor :packs
+
+    def initialize
+      @packs = {
+        simple_transportation_pack: {
+          intelligence: [:cellphone],
+          items: %i[medipack chevy_versa]
+        },
+        standard_transportation_pack: {
+          intelligence: %i[cellphone antenna],
+          arsenal: [:colt_1911],
+          items: %i[handcuffs medipack chemistry chevy_versa]
+        },
+        simple_mission_pack: {
+          intelligence: %i[infopack laptop cellphone antenna],
+          arsenal: [:colt_1911],
+          items: %i[handcuffs medipack chemistry financial]
+        },
+        standard_mission_pack: {
+          intelligence: %i[infopack laptop cellphone antenna],
+          arsenal: %i[remington_870 colt_1911 machete hatchet],
+          items: %i[handcuffs medipack chemistry financial]
+        }
+      }
+    end
+  end
+
+  class Control
+    attr_accessor :missions
+
+    def initialize
+      @missions = {}
+    end
+
+    def new_mission(name:, objective:, pack:)
+      @missions[name] = {
+        objective: objective,
+        pack: pack,
+        status: :active
+      }
```

Comment:
I would set these kind of hashes using one line if possible. `@missions[name] = { objective: objective, pack: pack, status: :active }`